autometrics-macros/src/result_labels.rs

```diff
@@ -94,21 +94,17 @@ fn conditional_label_clauses(
 fn extract_label_attribute(attrs: &[Attribute]) -> Result<Option<LitStr>> {
     attrs
             .iter()
-            .find_map(|att| match att.parse_meta() {
-                Ok(meta) => match &meta {
+            .find_map(|att| match &att.meta {
                     syn::Meta::List(list) => {
                         // Ignore attribute if it's not `label(...)`
                         if list.path.segments.len() != 1 || list.path.segments[0].ident != ATTR_LABEL {
                             return None;
                         }
 
                         // Only lists are allowed
-                        let pair = match list.nested.first() {
-                            Some(syn::NestedMeta::Meta(syn::Meta::NameValue(pair))) => pair,
-                            _ => return Some(Err(Error::new_spanned(
-                            meta,
-                            format!("Only `{ATTR_LABEL}({RESULT_KEY} = \"RES\")` (RES can be {OK_KEY:?} or {ERROR_KEY:?}) is supported"),
-                            ))),
+                        let pair = match att.meta.require_list().and_then(|list| list.parse_args::<syn::MetaNameValue>()) {
+                            Ok(pair) => pair,
+                            Err(err) => return Some(Err(Error::new_spanned(&att.meta, err))),
```

Comment:
is there a reason for changing the error message?

Reply:
Good catch! Forgot to change it back after some experiments